atom/browser/BUILD.gn

```diff
@@ -306,6 +308,8 @@ source_set("browser") {
   }
 
   if (enable_extensions) {
+    data_deps += [ "//electron:brave_resources" ]
```

Comment:
is this the right way to address your previous comment re: the missing `brave_resources.h` dependency? wasn't sure since it is generated and I'm not particularly familiar with GN